docs/requirements.md

```diff
@@ -67,3 +75,42 @@ Memory: 6 * (2 + (1 * 4)) = 36 GB
 CPU: 7 CPU + 18 CPU = 25 CPU
 Memory: 39 GB + 36 GB = 75 GB
 ```
+
+## PostgreSQL server setup
```

Comment:
Hopefully, by joining everything related to the database server setup here, it will be more apparent to customers what they need to do and that they need to be able to access the database server to run the script.